ts/Series/MapPoint/MapPointSeries.ts

```diff
@@ -211,10 +218,12 @@ class MapPointSeries extends ScatterSeries {
  * */
 
 interface MapPointSeries {
+    bounds: MapBounds | undefined;
     pointClass: typeof MapPointPoint;
 }
 extend(MapPointSeries.prototype, {
     type: 'mappoint',
+    bounds: MapSeries.prototype.bounds,
```

Comment:
Why? `MapSeries.prototype.bounds` is `undefined` at this stage. Since `bounds` is of type `MapBounds | undefined`, I think we can just delete this line.